modules/hal_nxp/usb/usb_device_config.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018 - 2022 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __USB_DEVICE_CONFIG_H__
+#define __USB_DEVICE_CONFIG_H__
+
+#include <zephyr/devicetree.h>
+#include "usb.h"
+
+/******************************************************************************
+ * Definitions
+ *****************************************************************************/
+/* EHCI instance count */
+#ifdef CONFIG_USB_DC_NXP_EHCI
+#define USB_DEVICE_CONFIG_EHCI (1U)
+/* How many the DTD are supported. */
+#define USB_DEVICE_CONFIG_EHCI_MAX_DTD (16U)
+/* Whether device is self power. 1U supported, 0U not supported */
+#define USB_DEVICE_CONFIG_SELF_POWER (1U)
```

Comment:
How is it used? There is the same definition in the line 42.

Reply:
I have deleted the duplicate define. This is used in the SDK driver.